spec/controllers/application_controller_spec.rb

```diff
@@ -157,4 +157,75 @@
       expect(assigns(:org_controller)).to eq("vm")
     end
   end
+
+  describe "#build_user_emails_for_edit" do
+    before :each do
+      EvmSpecHelper.local_miq_server
+      MiqUserRole.seed
+
+      role = MiqUserRole.find_by_name("EvmRole-operator")
+
+      group1 = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Group1")
+      group2 = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Group2")
+      group3 = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Group3")
+      current_group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Current Group")
+
+      @user1 = FactoryGirl.create(:user, :userid => "User1", :miq_groups => [group1], :email => "user1@test.com")
+      @user2 = FactoryGirl.create(:user, :userid => "User2", :miq_groups => [group2], :email => "user2@test.com")
+      @user3 = FactoryGirl.create(:user, :userid => "User3", :miq_groups => [group3], :email => "user3@test.com")
+
+      current_users_groups = [current_group, group1, group2]
```

Comment:
yes I agree, I reduced it, but I think we need to cover at least case when user belongs to more groups as I did it now. What do you it think ? or is it unnecessary ?